packages/recomponents/src/styles/typography.scss

```diff
@@ -0,0 +1,311 @@
+        url('/static/fonts/Montserrat-Regular.woff') format('woff'),
+        url('/static/fonts/Montserrat-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Montserrat';
+    src: url('/static/fonts/Montserrat-Medium.eot');
+    src: url('/static/fonts/Montserrat-Medium.eot?#iefix') format('embedded-opentype'),
+        url('/static/fonts/Montserrat-Medium.woff2') format('woff2'),
+        url('/static/fonts/Montserrat-Medium.woff') format('woff'),
+        url('/static/fonts/Montserrat-Medium.ttf') format('truetype');
+    font-weight: 500;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Montserrat';
+    src: url('/static/fonts/Montserrat-SemiBold.eot');
+    src: url('/static/fonts/Montserrat-SemiBold.eot?#iefix') format('embedded-opentype'),
+        url('/static/fonts/Montserrat-SemiBold.woff2') format('woff2'),
+        url('/static/fonts/Montserrat-SemiBold.woff') format('woff'),
+        url('/static/fonts/Montserrat-SemiBold.ttf') format('truetype');
+    font-weight: 600;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Montserrat';
+    src: url('/static/fonts/Montserrat-Bold.eot');
+    src: url('/static/fonts/Montserrat-Bold.eot?#iefix') format('embedded-opentype'),
+        url('/static/fonts/Montserrat-Bold.woff2') format('woff2'),
+        url('/static/fonts/Montserrat-Bold.woff') format('woff'),
+        url('/static/fonts/Montserrat-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+//Roboto Mono
+@font-face {
+    font-family: 'Roboto Mono';
+    src: url('/static/fonts/RobotoMono-Bold.eot');
+    src: url('/static/fonts/RobotoMono-Bold.eot?#iefix') format('embedded-opentype'),
+        url('/static/fonts/RobotoMono-Bold.woff2') format('woff2'),
+        url('/static/fonts/RobotoMono-Bold.woff') format('woff'),
+        url('/static/fonts/RobotoMono-Bold.ttf') format('truetype');
+    font-weight: bold;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto Mono';
+    src: url('/static/fonts/RobotoMono-Medium.eot');
+    src: url('/static/fonts/RobotoMono-Medium.eot?#iefix') format('embedded-opentype'),
+        url('/static/fonts/RobotoMono-Medium.woff2') format('woff2'),
+        url('/static/fonts/RobotoMono-Medium.woff') format('woff'),
+        url('/static/fonts/RobotoMono-Medium.ttf') format('truetype');
+    font-weight: 500;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto Mono';
+    src: url('/static/fonts/RobotoMono-Regular.eot');
+    src: url('/static/fonts/RobotoMono-Regular.eot?#iefix') format('embedded-opentype'),
+        url('/static/fonts/RobotoMono-Regular.woff2') format('woff2'),
+        url('/static/fonts/RobotoMono-Regular.woff') format('woff'),
+        url('/static/fonts/RobotoMono-Regular.ttf') format('truetype');
+    font-weight: normal;
+    font-style: normal;
+}
+
+h1, .h1 {
+    font-size: $h1;
```

Comment:
In the future, we'll want to allow customizing `font-size` and `font-weight` with CSS vars.